schema/elements.go

```diff
@@ -259,3 +263,26 @@ func (s *Schema) Resolve(tr TypeRef) (Atom, bool) {
 	}
 	return tr.Inlined, true
 }
+
+// Clones this isntance of Schema into the other
+// If other is nil this method does nothing.
+// If other is already initialized, overwrites it with this instance
+//
+// `dst` must not be shared across threads
+func (s Schema) CopyInto(dst *Schema) {
```

Comment:
is CopyInto really what we want, or is all of this a side-effect of having to dereference Schema in order to store a non-pointer in Parser? xref https://github.com/kubernetes/kubernetes/pull/109212#discussion_r840200723

Reply:
PR is updated to change schema to use pointers rather than adding `CopyInto`